src/lang/qbittorrent_zh_Hant.ts

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.0" language="zh_TW">
+<TS version="2.0" language="zh-Hant">
```

Comment:
Maybe this difference can be ignored if the blank QComboBox issue can be solved.